src/widgets/ListNftWidget/ListNftWidget.tsx

```diff
@@ -24,6 +26,7 @@ const ListNftWidget: FC<ListNftWidgetProps> = ({ className = '' }) => {
     && currencyTokenInfo
```

Comment:
`chainId` verification from this test (line15) is already included in verification for `chainIdIsCorrect`